File: genkin-web/src/main/resources/static/js/service/UserService.js
'use strict';

genkin
    .service('UserService', function($http) {
        this.signUp = function(user, okCb, errCb) {
            console.log("signUp: " + JSON.stringify(user));
            $http.post('/user/register', user, {}).then(okCb, errCb);
        };

        this.signIn = function(user, okCb, errCb) {
            console.log("signIn: " + JSON.stringify(user));
            $http.post('/user/login', user, {}).then(okCb, errCb)
        };
    });

File: genkin-server/src/main/java/dai/genkin/genkinserver/service/UserService.java
package dai.genkin.genkinserver.service;

import dai.genkin.genkinserver.dao.UserRepository;
import model.User;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

/**
 * ionutciuta24@gmail.com on 18.08.2017.
 */
@Service
public class UserService {

    @Autowired
    private UserRepository userRepository;

    public User register(User user) {
        return userRepository.save(user);
    }

    public User login(User user) {
        return userRepository.save(user);
    }
}
